README.md

```diff
@@ -1 +1,6 @@
-# hello-world
\ No newline at end of file
+Hello microverse
+
+its my first project in microverse
+
+
+the aim is to learn the github basics and other microverse professional standards
```

Comment:
- [ ] The project should have a descriptive README.md ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)).
A good README should have a description that is clear, short, and to the point. It should describe the importance of your project, what it does and how to use it. This way technical and non-technical users can understand how to use it.  Refer to this template [here](https://github.com/microverseinc/readme-template) and customise it to your needs.